src/services/side-panels/types/types.ts

```diff
@@ -0,0 +1,3 @@
+export type SidePanelPosition = 'SidePanelTop' | 'SidePanelLeft' | 'SidePanelRight' | 'SidePanelBottom';
```

Comment:
doesn't this make getPanelArea redundent?

Reply:
i want back to what it was before